web/js/map/natural-events/event-markers.js

```diff
@@ -114,24 +118,22 @@ class EventMarkers extends React.Component {
         ? category
         : { title: 'Default', slug: 'default' };
 
-      // get maxExtent of current projection and check if marker is within range
-      const maxExtentCheck = olExtent.containsCoordinate(maxExtent, coordinates);
-      // only create marker if within projection extent range
-      if (maxExtentCheck) {
-        marker.pin = createPin(event.id, category, isSelected, event.title);
-        marker.pin.setPosition(coordinates);
-        map.addOverlay(marker.pin);
-        this.addInteractions(marker, event, date, isSelected);
-      }
+      marker.pin = createPin(event.id, category, isSelected, event.title);
```

Comment:
It appears some event markers are being displayed as double: the selected (larger) and unselected (smaller) markers.

http://localhost:3000/?v=-106.9898766383032,40.041858413362824,-106.62430656994383,40.27174732937845&e=EONET_5432,2021-06-20&efs=false&efd=2016-02-29,2021-06-23&efc=dustHaze,manmade,seaLakeIce,severeStorms,snow,volcanoes,waterColor,wildfires&l=VIIRS_SNPP_Thermal_Anomalies_375m_Night(hidden),VIIRS_SNPP_Thermal_Anomalies_375m_Day(hidden),MODIS_Aqua_Thermal_Anomalies_Night(hidden),MODIS_Aqua_Thermal_Anomalies_Day(hidden),VIIRS_NOAA20_Thermal_Anomalies_375m_Night(hidden),VIIRS_NOAA20_Thermal_Anomalies_375m_Day(hidden),MODIS_Terra_Thermal_Anomalies_Night,MODIS_Terra_Thermal_Anomalies_Day,VIIRS_SNPP_DayNightBand_ENCC(hidden),VIIRS_SNPP_DayNightBand_At_Sensor_Radiance(hidden),Reference_Labels_15m,Reference_Features_15m,Coastlines_15m(hidden),VIIRS_SNPP_CorrectedReflectance_BandsM11-I2-I1(hidden),VIIRS_SNPP_CorrectedReflectance_TrueColor(hidden),MODIS_Aqua_CorrectedReflectance_Bands721(hidden),MODIS_Aqua_CorrectedReflectance_TrueColor(hidden),VIIRS_NOAA20_CorrectedReflectance_BandsM11-I2-I1(hidden),VIIRS_NOAA20_CorrectedReflectance_TrueColor(hidden),MODIS_Terra_CorrectedReflectance_Bands721(hidden),MODIS_Terra_CorrectedReflectance_TrueColor&lg=true&t=2021-06-21-T00%3A00%3A00Z

Reply:
This definitely sounds like something that would happen if the markers weren't being removed when switching away from the events tab (which is what was happening in some cases).  Since that should be fixed now, I'm assuming this will be too since I don't see the issue when loading this link directly.